comp/ops/compstate.yaml

```diff
@@ -9,22 +9,28 @@ milestone: $SRYYYY Competition
 area-owner: game
 
 description: >-
-    The competition software has state about the competition which is used to
-    drive the matches, live displays of scores (etc.) both within the venue on
-    the website.
+  The competition software has state about the competition which is used to drive the matches,
+  live displays of scores (etc.) both within the venue and on the website.
 
-    This task is to create the initial state for the competition; see
-    https://github.com/PeterJCLaw/srcomp/wiki/Competition-Preparation#create-a-compstate
-    and https://github.com/search?q=topic%3Acompstate+org%3Asrobo&type=Repositories.
+  This task includes creating the initial state for the competition; see
+  https://github.com/PeterJCLaw/srcomp/wiki/Competition-Preparation#create-a-compstate
+  and https://github.com/search?q=topic%3Acompstate+org%3Asrobo&type=Repositories.
+
+  Alongside this, the format for the game data YAML files which record match outcomes and the script used to calculate the scores needs to be designed.
+  There is guidance for how to do the former for SRComp at
+  https://github.com/PeterJCLaw/srcomp/wiki/Competition-Preparation#configure-game-scoring
+  See https://github.com/PeterJCLaw/srcomp/wiki/Competition-Preparation#implement-scorepy for details of the latter.
+
+  To complete the compstate we need to import the teams and schedule data.
+  This may also need to be done again at the competition.
+
+  Required actions:
```

Comment:
I would recommend that we keep these as separate tasks. There are lots of separate dependencies to parts of these and many of them can be completed in isolation while others can't. For example: it's hard to define the YAML schema without also doing most of the work on the scoring logic (and also the physical scoring form), however those are very much independent from the day & match schedules. The latter depend on `comp/game/match-schedule`, yet that's no reflected here (and it would be a bit odd to imply that everything here depends on `comp/game/match-schedule` when that's not really the case either).

Reply:
The intention here was to collect together the steps requred to setup a compstate. In general task dependencies are scoped more on what needs to be completed to complete the overall task rather than each individual piece. 

As we're using GitHub task lists it's still possible to split the actions into their own issues (with a single click!) so if an area owner wants more granularity they can have it. By having a single task, it is easier to see what is left to do and to track progress.

Reply:
I see. My concern is that this was how the previous iteration of meta (ish) tasks ended up and yet those were also the most complained-about of the task types.
The refactoring in this PR seems to both be throwing away the more nuanced dependency information which was previously available at the same time as creating more of these harder-to-track tasks.

I would disagree about the ease of breaking out the sub-tasks. While, yes, a new issue can be created with a single click, the resulting issue has little to no information on it (no instructions, no dependencies, no cross linking). One of the core goals of the recurring tasks system was to create a place where these nuances could be captured to improve processes in the future. While this PR has done a great job of bringing the content up to date with what we do, it's not clear to me that the resulting structure ensures clarity around the dependencies of the tasks in question.

Consider for example the consequences of the compressed structure proposed in this PR:
- the virtual competition depends on the compstate (not sure if this is represented currently, if no it should be)
- the compstate task (here) depends on the physical competition day- and match-schedules
- thus the virtual competition _appears_ to depend on the physical competition day- and match-schedules

Yet we know from experience that this is not true -- I don't think we've had the physical competition day- and match-schedules ready before the virtual competition in the past, yet that hasn't and wouldn't affect anything.